### ISSUE TITLE
Custom Baritone build for Impact 4.8.3 doesn't work

### ISSUE THREAD

**Issue by daedsidog** [cabaletta/baritone#1348]
I built the current version of baritone using the command line guide, and got only 3 artifacts inside `libs`, one of which was `baritone-1.2.11.jar` not (`baritone-api-1.2.11.jar` as the guide mentions). I replaced the files in the MC `libraries` folder, and deleted the entire body of Impact's JSON to look like this:

![image](https://user-images.githubusercontent.com/41439659/75617769-4b87ad80-5b6c-11ea-8fe2-a35f77a6f174.png)

I have the Forge version of Impact, and when I ran `.b version`, it said I was using a null version. None of the new commands, such as `.b buildIgnoreExisting` were working, and even if I completely delete the new `.jar` from the `libraries` directory, it behaves exactly the same.

TLDR: How exactly do I update Baritone for Impact 4.8.3?

**Comment by scorbett123** [comment]
Closing as this is an issue for impact rather than baritone.
